setup.py

```diff
@@ -37,11 +37,9 @@
     # TODO: remove pyopencl when dpctl exposes preferred_work_group_size_multiple
     # see https://github.com/IntelPython/dpctl/issues/886
     install_requires=["scikit-learn", "numba-dpex", "dpctl", "pyopencl"],
-    extra_requires=[dict(benchmark=["scikit-learn-intelex, pandas"])],
+    extra_require=[dict(benchmark=["scikit-learn-intelex, pandas"])],
```

Comment:
Thanks for fixing this! I could not understand what was wrong myself!

Reply:
An extra character and the world is broken. :upside_down_face:

I find the inconsistency in the naming of `install_requires` and `extra_require` weird and error-prone as shown here.